tests/Persistence/SqlAllMappingsLookupTest.php

```diff
@@ -0,0 +1,230 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace ProfessionalWiki\WikibaseRDF\Tests\Persistence;
+
+use ProfessionalWiki\WikibaseRDF\Application\EntityMappingList;
+use ProfessionalWiki\WikibaseRDF\Application\Mapping;
+use ProfessionalWiki\WikibaseRDF\Application\MappingList;
+use ProfessionalWiki\WikibaseRDF\Persistence\SqlAllMappingsLookup;
+use ProfessionalWiki\WikibaseRDF\WikibaseRdfExtension;
+use Wikibase\DataModel\Entity\EntityId;
+use Wikibase\DataModel\Entity\Item;
+use Wikibase\DataModel\Entity\ItemId;
+use Wikibase\DataModel\Entity\Property;
+use Wikibase\DataModel\Entity\PropertyId;
+use Wikibase\Repo\WikibaseRepo;
+
+/**
+ * @covers \ProfessionalWiki\WikibaseRDF\Persistence\SqlAllMappingsLookup
+ * @group Database
+ */
+class SqlAllMappingsLookupTest extends \MediaWikiIntegrationTestCase {
```

Comment:
This class feels a bit bulky at the moment. The object creation and testing should be optimized in some way.

Reply:
I added Items and Properties. Is this something we should test explicitly since we support both, even though they have the same base class? I'm not sure if they have some other internal differences that would be safer to test explicitly.

Reply:
As far as this code is concerned items and properties are the same